entries/skyland/game.json

```diff
@@ -5,14 +5,14 @@
             "default": true,
             "filename": "Skyland.gba",
             "hash": {
-                "md5": "e4cb9e6400645e7ddd55005e5013a804",
-                "sha1": "d8996aa4f6c7c7be5f0eec3df5466c717a3d5a95",
-                "sha256": "f5f27fa1e01fc27a24f8c831b749ba5f1c8c04a3b05e9a98a1bd9bea72cef93e"
+                "md5": "498882021ae28f6c5e3d453509d4e6c8",
+                "sha1": "6c5eba5faf9a54c4af05ecd6c1fbfaf9be0417e3",
+                "sha256": "01c0a4fd2fe56e7563f84b3be4b61331f1cace98c8392f14e2a53fd5d9f34784"
             },
             "playable": true
         }
     ],
-    "license": "MIT",
+    "license": "Custom",
```

Comment:
I'm surprised this passes the schema validator, to be honest :/
Anyway, if I understood correctly now the game is basically "source-available" but not open source anymore so we can simply remove the "license field"? Or maybe ad a "source-available" option for it..

Reply:
I unfortunately had to change the license after dealing with issues with pirated bootleg cartridges, and licensing complaints from Steam, who do not like open source licenses that much. The MIT license, which applied to the jam version of the game, was later superseded by the GPL license, which I used until valve rejected my game submission. I would be fine with either removing the license field or labeling it as source available, let me know which you'd prefer and I'll fix my PR

Reply:
Shit, that sucks. I'd say, for the sake of keeping this PR in scope, removing the `Open Source` tag and the `license` field should be okay and we can merge it quickly. I can in parallel open an issue regarding changing the Schema to accomodate and expose "source-available" entries faithfully.

Reply:
While it's no longer open source, the current license allows modification and redistribution, the only limitation is on selling copies of the ROM. The license is still compatible with publishing on homebrew hub :)

Reply:
Yep, sure! But the entry shouldn't be listed there as "Open Source" anymore, no ?

Reply:
Yeah correct! I tried to update the checksums, the license field, and the other stuff in the game.json, please let me know if there's anything I can do to fix my PR, I am happy to make any changes that you suggest!

Reply:
Maybe by github ui is glitched? I see the license field still there (which we can remove) and the "Open Source" tag still in the tags?

Reply:
(I'd love to have "source available" / "custom" as possible values in "license" but the schema is not ready for it and pushing the entry as "custom" license would break the frontends, which atm expect SPDX identifiers..)